### PR TITLE
hc bRanD reFrEsH

### PR DESCRIPTION
**Pull request by jackyzha0**
* made some colours more consistent
* hc theme yeet

Screen Shot 2020-11-14 at 4 06 38 PM

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -16,10 +17,19 @@ const SidebarContainer = styled.div`
...
+const Logo = styled.img.attrs(p => ({
+  src: p.theme.custom_imgs === 'hc' ? hc_logo : logo,
+}))`
   width: 80px;
   height: 85px;
   margin: 30px 0 0px 50px;
+
+  ${p =>
+    p.theme.custom_imgs === 'hc' &&
+    css`
```
Comment (ianmah) [inline src/components/Sidebar.js:29]: why do we need this?

Comment (jackyzha0) [reply src/components/Sidebar.js:29]: HC logo is more rectangle than square, so just fixing width and padding to make it look nicer

Comment (ianmah) [reply src/components/Sidebar.js:29]: You can't do that without `css`?

Comment (jackyzha0) [reply src/components/Sidebar.js:29]: how else should i do the conditional css bit?

Comment (ianmah) [reply src/components/Sidebar.js:29]: does this not work
```
  ${p =>
    p.theme.custom_imgs === 'hc' &&
    `
      width: 120px;
      margin: 30px 0 0px 60px;
    `}
`
```

Comment (jackyzha0) [reply src/components/Sidebar.js:29]: lemme try that

File: src/components/Common.js [hunk 2]
```diff
@@ -37,6 +38,14 @@ export const Button = styled.a`
   padding: 0.75em 2.5em;
   border-radius: 3px;
   margin: 1em;
+
+  ${p =>
+    p.theme.custom_imgs === 'hc' &&
+    css`
```
Comment (ianmah) [inline src/components/Common.js:44]: same thing here?

**Review by ianmah — APPROVED** [review]

File: src/components/Common.js (hunk 2 above)
Comment (jackyzha0) [reply src/components/Common.js:44]: this file already imports css so i think it should be fine lol

**Pull request merged**